src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/arm/reorder_deconv_nhwc.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_test_utils/node_builders/constant.hpp"
+#include "ov_models/utils/ov_helpers.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "utils/cpu_test_utils.hpp"
+#include "common_test_utils/node_builders/convolution_backprop_data.hpp"
+
+using namespace CPUTestUtils;
+
+namespace ov {
+namespace test {
+
+// Subgraph:
+/*
+┌──────────────────┐           ┌──────────────────┐
+│      INPUT       │           │      WEIGHTS     │
+└─────────┬────────┘           └─────────┬────────┘
+          │                              │
+┌─────────┴────────┐                     │
+│  REORDER (NHWC)  │         NO REORDER  X
+└─────────┬────────┘                     │
+          │      ┌──────────────────┐    │
+          └──────┤  DECONVOLUTION   ├────┘
+                 └─────────┬────────┘
+                           │
+                 ┌─────────┴────────┐
+                 │  REORDER (NCHW)  │
+                 └─────────┬────────┘
+                           │
+                 ┌─────────┴────────┐
+                 │      OUTPUT      │
+                 └──────────────────┘
+ */
+
+class ReorderDeconvNHWCTest : virtual public SubgraphBaseStaticTest {
```

Comment:
Could you please provide details why we need this test?

Reply:
@dmitry-gorokhov we test it because in ACL executor for deconvolution layer we have transpose and we remove reorder for nhwc and move functionality in transpose (and that it we test)